app/models/comment.rb

```diff
@@ -3,5 +3,9 @@
 class Comment < ApplicationRecord
   belongs_to :user
   belongs_to :review
-  belongs_to :parent_comment
+  belongs_to :parent_comment, class_name: "Comment", optional: true
```

Comment:
```suggestion
  belongs_to :parent_comment, class_name: Comment.name, optional: true
```
hỏi dzuiii optional true để làm chi thế em

Reply:
Em nhớ hồi đợt phỏng vấn chị có nói về join self, em cũng đang áp dụng, nhưng mà nếu join self thì sẽ bị ràng buộc foriegnKey -> foriegnKey not null -> nó sẽ xảy ra vấn đề là first comment sẽ không cần ( không có ) foriegnKey -> nếu không có option thì sẽ có lỗi missing foriegnKey ạ 😅

Reply:
@vanvtt-0952